bitnami/apache/values.yaml

```diff
@@ -121,13 +121,17 @@ resources:
 ##
 livenessProbe:
   enabled: true
+  path: "/"
+  port: http
   initialDelaySeconds: 180
   periodSeconds: 20
   timeoutSeconds: 5
   failureThreshold: 6
   successThreshold: 1
 readinessProbe:
   enabled: true
+  path: "/"
+  port: http
```

Comment:
These new parameters should be listed in the README.md

Reply:
new path parameter added as well as readinessProbe.enabled and livenessProbe.enabled